web/src/utilities/utilities.js

```diff
@@ -700,8 +700,11 @@ export const Utilities = {
       window.localStorage.removeItem("session_roles");
     }
 
-    if (window.location.pathname !== "/secure-console") {
-      window.location = "/secure-console";
+    const redirectPath = options?.snapshotRestore
+      ? "/restore-completed"
+      : "/secure-console";
+    if (window.location.pathname !== redirectPath) {
+      window.location = redirectPath;
```

Comment:
Kots is not set up to go anywhere but the login page when the user is not authed. This is a hacky way to support another route. Basically if the user is logged out it will redirect to the /restore-complete page when options.snapshotRestore is passed in.

Future state I think this should be in a top level routes definition file similarly to how BPR does it. The top level routes file uses any state necessary to make routing decisions like useAuth in this example:

https://github.com/alan2207/bulletproof-react/blob/master/src/routes/index.tsx

We have similar functionality but we need to isolated it like this.